packages/pointer_interceptor/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
-## 0.9.2
+## 0.9.3
 
 * Marked `PointerInterceptor` as invisible, so it can be optimized by the engine.
```

Comment:
This shouldn't be moved from 0.9.2 to 0.9.3. It was in 0.9.2, and there are ways to get 0.9.2 (force-pinning, using old versions of Flutter).